cmd/tealdbg/debugger_test.go

```diff
@@ -130,15 +130,17 @@ func createSessionFromSource(t *testing.T, program string) *session {
 
 	// create a sample disassembly line to pc mapping
 	// this simple source is similar to disassembly except intcblock at the beginning
-	pcOffset := make(map[int]int, len(ops.OffsetToLine))
-	for pc, line := range ops.OffsetToLine {
-		pcOffset[line+1] = pc
+	offsetToLine := make(map[int]int, len(ops.OffsetToSource))
```

Comment:
Since you're exporting the SourceLocation anyway, it seems just as easy to change `programMeta` and `session` in `debugger.go` to use the enhanced `OffsetToSource` map rather than build up a new one that only maps lines to keep them happy.

Reply:
This change would be a more invasive than I'd like to be, since the `tealdbg` tool has its own legacy version of a source map (see `session.GetSourceMap`), which has its own quirks. I don't think there's a ton of test coverage here, so I'd rather just leave it be.

Reply:
This seems safe, and would give us columns in that source map.
https://github.com/jasonpaulos/go-algorand/pull/3/files